lightning/src/onion_message/functional_tests.rs

```diff
@@ -34,6 +34,8 @@ use core::ops::Deref;
 
 use crate::prelude::*;
 
+use core::ops::Deref;
```

Comment:
Little confused about 4cab9cbcb29308a47efcfcfbfed186211cc48848. The message doesn't seem to match the change and then it is reverted in the next commit.